doc/usage.rst

```diff
@@ -361,42 +361,42 @@ to reproduce the version text file.
 Building Mac OS X App Bundles
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you specify only ``--onefile`` under Mac OS X, the output
-in :file:`dist` is a UNIX executable
-:file:`myscript`.
-It can be executed from a Terminal command line.
-Standard input and output work as normal through the Terminal window.
-
-If you also specify ``--windowed``, the ``dist`` folder contains
-two outputs: the UNIX executable :file:`myscript`
-and also an OS X application named :file:`myscript.app`.
+Under Mac OS X, |PyInstaller| always builds a UNIX executable in
+:file:`dist`.
+If you specify ``--onedir``, the output is a folder named :file:`myscript`
+containing supporting files and an executable named :file:`myscript`.
+If you specify ``--onefile``, the output is a single UNIX executable
+named :file:`myscript`.
+Either executable can be started from a Terminal command line.
+Standard input and output work as normal through that Terminal window.
+
+If you specify ``--windowed`` with either option, the ``dist`` folder
+also contains an OS X application named :file:`myscript.app`.
```

Comment:
"also"? This means in addition to the file when using --console? May not be worth changing the text, I'm just curious.

Reply:
Yes, it builds the "normal" output (dir or file) and also builds the myscript.app.